Description: Instalss the application icon in the correct path
 Application icon is 32x32: install it in /usr/share/icons/hicolor/32x32/apps
 instead of in /usr/share/icons/hicolor/48x48/apps
Author: Leonardo Montecchi <lmontecchi@montex.org>
Last-Update: 2014-01-25

--- luminance-hdr.orig/CMakeLists.txt
+++ luminance-hdr/CMakeLists.txt
@@ -270,7 +270,7 @@
     INSTALL(FILES ${LUMINANCE_QM} DESTINATION share/luminance-hdr/i18n) # i18n
     INSTALL(FILES ${LUMINANCE_QT_QM} DESTINATION share/luminance-hdr/i18n) # i18n
 
-    INSTALL(FILES ${CMAKE_SOURCE_DIR}/images/luminance-hdr.png DESTINATION share/icons/hicolor/48x48/apps) # icon
+    INSTALL(FILES ${CMAKE_SOURCE_DIR}/images/luminance-hdr.png DESTINATION share/icons/hicolor/32x32/apps) # icon
     INSTALL(FILES ${CMAKE_SOURCE_DIR}/luminance-hdr.desktop DESTINATION share/applications) #desktop entry
     INSTALL(FILES
     ${CMAKE_SOURCE_DIR}/AUTHORS
